.github/workflows/pep8.yml

```diff
@@ -0,0 +1,54 @@
+name: Python
```

Comment:
Hi, this file already exists as `pep8.yaml`, can you please remove it from your PR?